$OpenBSD: patch-configure,v 1.2 2008/12/14 18:46:46 ajacoutot Exp $

There is no --disable-pulseaudio switch, but we don't want to pick it up.
So forse it the hard way.

--- configure.orig	Tue Nov 25 00:21:30 2008
+++ configure	Sun Dec 14 17:36:33 2008
@@ -22915,7 +22915,7 @@ fi
 # ---------------------------------------------------------------------------
 # Enable Legacy Sound Preference
 # ---------------------------------------------------------------------------
-
+if false; then # Don't just pickup pulseaudio.
 pkg_failed=no
 { $as_echo "$as_me:$LINENO: checking for PULSEAUDIO" >&5
 $as_echo_n "checking for PULSEAUDIO... " >&6; }
@@ -22982,6 +22982,10 @@ else
 $as_echo "yes" >&6; }
 	have_pulseaudio=yes
 fi
+
+fi # Don't just pickup pulseaudio.
+have_pulseaudio=no
+
 if test x$have_pulseaudio = xyes; then
 	default_enable_legacy_sound_pref=no
 else
